.github/ISSUE_TEMPLATE/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Engineering Manager Onboarding
+about: A list of things to do as a new manager to become familiar with the nature of our engineering team and the context it exists in.
+---
+
+## After your first week
+
+You’ve done general people-ops and engineering team onboarding, and have casually met some team-members. Now it’s time to start onboarding as a new engineering manager.
+
+- [ ] If you’re reading this, you’re the first external engineering manager we’ve hired and your name is Adam. Yay! (If not, this will be _real_ awkward.)
+
+     At Artsy we are trying to give new hires ownership over improving the onboarding documentation that they are faced with, in an effort to help the next new hire onboard more smoothly. As you are the first one, there is plenty of work to do in this area; in fact, this very document is nothing but a first draft. Things in this document may turn out to be insufficient, too broad, or simply not work at all. That’s all good and you should feel empowered to bring this up openly.
+
+     It will be your task to make this document into a substantially larger one, that draws from all the things you had to learn. You’re not on your own, your fellow peers will all be available to support you during this task. You got this!
+
+- [ ] The people-ops team should have signed you up for the various on-site manager trainings.
+
+- [ ] Your manager should have invited you to meetings relevant to managers.
+
+- [ ] Your manager should have scheduled 1:1s for you with the other engineering managers, including the CTO, and other leadership people.
+
+## Your first ~two months
+
+The goal of onboarding is to set you up for success as a new engineering manager at Artsy. To that end, your main objective during this time will be to gather knowledge. You will be given ample time to get acquainted with your new peers, our business, and our tech.
+
+_For now we’ll try to target roughly two months, as there are currently 5 product teams and sprints have a duration of 2 weeks. Whether or not this duration was too arbitrary is something we’ll have to learn._
+
+To help work on the main objective, we’ve tried to identify the following goals.
+
+### Get to know Artsy's team
+
+- [ ] Have 1:1s with each and every engineer (that includes tech-leads) to get to know the people, their interests, their north-star goals, and their perspectives on the businesses they contribute to.
```

Comment:
as a possible suggestions, would it be easier if these 1:1s where setup for each team members of the current rotation? Like if you are in Sell this Sprint, setup 1:1s with all sell team members during this sprint and etc.

Reply:
we could also suggest few famous Artsy 1:1s places if Adam is not familiar with the neighborhood :)

Reply:
I think the other person in each of his 1:1s will probably show him around the neighborhood. But if there are specific advantages for choosing some places over others for one’s 1:1, then that would be helpful.